data/common.yaml

```diff
@@ -34,3 +38,32 @@ systemd::manage_logind: false
 systemd::logind_settings: {}
 systemd::manage_all_network_files: false
 systemd::network_path: '/etc/systemd/network'
+systemd::loginctl_users: {}
+systemd::dropin_files: {}
+systemd::udev_rules: {}
+
+lookup_options:
```

Comment:
This is not a standard pattern and would caution against specifying the lookup strategy. Usually this is not needed and if it is, that is managed outside a module.